test/common/testlib.py

```diff
@@ -204,7 +204,7 @@ def open(self, href, cookie=None, tls=False):
             schema = tls and "https" or "http"
             href = "%s://%s:%s%s" % (schema, self.address, self.port, href)
 
-        if not self.current_layout:
+        if not self.current_layout and os.environ.get("TEST_SHOW_BROWSER") in [None, "pixels"]:
```

Comment:
This is just the initial size, calls to assert_pixels and set_layout will still resize the browser.  Is that acceptable?

Reply:
That's debatable, I guess -- between YAFIYGI or just ignoring the assert_pixels() calls with TEST_SHOW_BROWSER, but I currently tend towards the former. At least it doesn't get in the way so much now.